From 6635067a209bb887a4382d393282d9cb745a0ece Mon Sep 17 00:00:00 2001
From: yvanliang <yvan6liang@outlook.com>
Date: Fri, 9 Oct 2026 15:43:46 +0800
Subject: [PATCH 04/11] GSI Android 17 compatibility:
 0004-Don-t-abort-in-case-of-cgroup-bpf-setup-fail-since-s.patch

Change-Id: Ie676239f2ba17e121f73d1bfa09f6075d217db6e
---
 server/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/main.cpp b/server/main.cpp
index be9a1d0..e84899e 100644
--- a/server/main.cpp
+++ b/server/main.cpp
@@ -147,7 +147,7 @@ int main(const int argc, const char* argv[]) {
 
     if (libnetd_updatable_init(cg2_path.c_str())) {
         ALOGE("libnetd_updatable_init failed");
-        exit(1);
+        //exit(1);
     }
     gLog.info("libnetd_updatable_init success");
 
